pkgs/applications/misc/netsurf/default.nix

```diff
@@ -0,0 +1,80 @@
+{ stdenv, fetchurl, SDL, pkgconfig, libpng, openssl, curl, gtk2, check
+, libxml2, libidn, perl, nettools, spidermonkey_185, nspr, perlPackages
+, netsurf-buildsystem
+, netsurf-nsgenbind
+, netsurf-libnsfb
+, netsurf-libwapcaplet
+, netsurf-libparserutils
+, netsurf-libcss
+, netsurf-libhubbub
+, netsurf-libdom
+, netsurf-libnsbmp
+, netsurf-libnsgif
+, netsurf-libnsutils
+, netsurf-libutf8proc
+}:
+
+stdenv.mkDerivation rec {
+
+  name = "netsurf-${version}";
+  version = "3.5";
+
+  # UIS incldue Framebuffer, and gtk, but
```

Comment:
How buggy is "buggy"? Is it worth it to add this complexity to support a broken feature?